src/dotrun_docker/models.py

```diff
@@ -161,14 +161,15 @@ def exec(self, commands, exit_on_error=True, background=False):
             self.env["PATH"] = self.pyenv_path + "/bin:" + self.env["PATH"]
             self.env.pop("PYTHONHOME", None)
 
-            if not os.path.isfile(f"{self.pyenv_path}/bin/python3.8"):
+            # This hard requirement can be removed once other python versions
+            # have been tested with dotrun on canonical web projects
+            if not os.path.isfile(f"{self.pyenv_path}/bin/python3.10"):
```

Comment:
I'm not sure I understand the reason behind this check, this code is running inside the container right, so the python3 version is specified by the `Dockerfile` setup. The user has no ability to change the python3 version in this scenario.

Reply:
The check exists because dotrun creates a virtualenv in the working directory and runs python commands through it.
The check then really should say: dotrun only supports python x, but your virtualenv at `.venv` is using python y.
I'll update it!

Reply:
Thanks Sam, this is clearer!